python/lsst/ap/association/diaPipe.py

```diff
@@ -214,9 +217,12 @@ class DiaPipelineConfig(pipeBase.PipelineTaskConfig,
         dtype=str,
         default="deep",
     )
-    apdb = daxApdb.ApdbSql.makeField(  # TODO: remove on DM-43419
+    apdb = pexConfig.ConfigurableField(  # TODO: remove on DM-43419
+        target=daxApdb.ApdbSql,
```

Comment:
I don't think this is necessary. I can retarget `apdb` to use `ApdbCassandra` without this change, and I don't imagine we'll be using the old system for Cassandra after this is merged.

Reply:
I don't understand this comment. `target` is a mandatory parameter for `ConfigurableField`; you can't leave it out.

Reply:
To clarify my comment: is this change to support the new config system, or the old one? If it's for the new system then I retract my statement, but if it's to support the old system then I would argue that it is not necessary.

Reply:
Can you maybe specify exactly what you mean by "this change"? Surely you're not objecting to the deprecation warning?

Reply:
`apdb = daxApdb.ApdbSql.makeField` -> `apdb = pexConfig.ConfigurableField`

Reply:
`makeField` does not support `deprecated`.

Reply:
OK, I didn't know that. Sorry for the noise!